lookout/style/typos/corrector_manager.py

```diff
@@ -25,7 +27,7 @@ def get(self, source: str) -> TyposCorrector:
         """
         obj = self.correctors.get(source)
         if obj is None:
-            obj = TyposCorrector().load(source)
+            obj = TyposCorrector().load(source, backend=modelforge.backends.create_backend())
```

Comment:
We need it to make it work if `source` is some part of  `UUID`, right?

Reply:
Yes, it should work with UUIDs. Actually, I need to edit Modelforge to create the backend inside, so this will not be needed.